int32 startX
int32 startY
int32 width
int32 height
---
bool status

